get_projects.json

```diff
@@ -0,0 +1,18 @@
+{
+  "operationName": "GetProjectsQuery",
+  "variables": {
+    "input": {
+      "cursor": "",
+      "filter": {
+        "teamId": "1",
+        "tags": ["TAG_NAME"]
```

Comment:
this one should be the auto generated ID, right?

Reply:
yes, but to make it easier so user don't have to query team tags to get the ID, I make it so that user can just input tag name instead

Reply:
didn't realize the `get_team_tags`, nice one